src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -479,15 +480,12 @@ trait Implicits { self: Typer =>
       formal.argTypes match {
         case arg :: Nil =>
           val tp = fullyDefinedType(arg, "ClassTag argument", pos)
-          tp.underlyingClassRef(refinementOK = false) match {
```

Comment:
It's not directly comparable. My point is the new implementation has some justification (stable erasure) where the old one hadn't.